src/initialization/swaggerSettings.js

```
@@ -0,0 +1,23 @@
+const swaggerJsDoc = require('swagger-jsdoc')
+
+const swaggerOptions = {
+  definition: {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)